applications/FluidDynamicsApplication/custom_elements/compressible_navier_stokes_explicit_2D3N.cpp

```diff
@@ -41,6 +41,12 @@
 
 namespace Kratos {
 
+template<>
+GeometryData::IntegrationMethod CompressibleNavierStokesExplicit<2,3>::GetIntegrationMethod() const
+{
+    return GeometryData::IntegrationMethod::GI_GAUSS_1;
```

Comment:
Correct me if I'm wrong, but isn't this element the one with "pre-computed" integration in the automatic differentiation? If so it is, I'd throw an error in here, as this method is expected to be never called.

Reply:
OK. Now I see, you use it for the auxiliary values. Fine then.

Reply:
Maybe we can put a comment saying that this is not used in the LHS and RHS evaluation.

Reply:
It is not used during integration but it is used in `CalculateMidPoint*` methods.

Before, we simply called `GI_GAUSS_1` directly. I moved it to this method to make it coherent with other elements.

I see the implementation in these midpoint methods assumes the shape function gradients to be uniform, so I guess there is some danger here since this mehtod gives the false impression of allowing change of integration method.

I can revert to simply using `GI_GAUSS_1` in place or remove the assumption for middle point computation (just like the quad element, so it would take no effort to change). Shouldn't really affect performance since we only need the shape function at the centerpoint.

Reply:
Oh ok you noticed before I finished writing 😅

Reply:
No no, I'm fine with this. However, I'd put a comment in line 47 saying that this is only use for the postprocessing of auxiliary variables, no in the LHS and RHS calculation (just to avoid future misunderstandings).